conf/ListOfTestCases.txt

```diff
@@ -1,4 +1,4 @@
 data/soapvalidator/Admin/Domains/domains_get.xml
 data/soapvalidator/Admin/AdminSearch/Aliases
 data/soapvalidator/MailClient/Item
-data/soapvalidator/MailClient/Mail/message_action.xml
```

Comment:
data/soapvalidator/MailClient/Mail/message_action.xml - why is this test skipped?

Reply:
This has been done as per Previous PR suggestions like "I don't think we need to change legacy edition of this file." by Swapnil. 

So i have deleted all skip files of Zimbra X specially STAF related and copied from Develop branch.